CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.1 (2021-11-09)
+
+- Fixed toolbar disabled button styles in modern theme. [#261](https://github.com/blackbaud/skyux-layout/pull/261)
```

Comment:
```suggestion
- Fixed the toolbar disabled button styles in modern theme. [#261](https://github.com/blackbaud/skyux-layout/pull/261)
```
Missing an article, but there's probably a better way to write this. I'll let Docs chime in...

Reply:
```suggestion
- Fixed the toolbar's disabled button styles in modern theme. [#261](https://github.com/blackbaud/skyux-layout/pull/261)
```
Not much to add except that I'd also make "toolbar" possessive.